src/MoBi.Presentation/Presenter/Main/ExplorerPresenter.cs

```diff
@@ -167,5 +185,16 @@ private static int nameComparison(ITreeNode<IWithName> a, ITreeNode<IWithName> b
             return string.Compare(a.Tag.Name, b.Tag.Name, StringComparison.InvariantCultureIgnoreCase);
          return 0;
       }
+
+      private static int orderFor(BuildingBlock buildingBlock)
+      {
+         if (buildingBlock == null)
+            return int.MaxValue;
+
+         if (_buildingBlockOrder.TryGetValue(buildingBlock.GetType(), out var order))
+            return order;
+
+         return int.MaxValue;
```

Comment:
Then you can just go

```
         if (buildingBlock == null)
            return int.MaxValue;

         return _buildingBlockOrder[buildingBlock.GetType()];
```

because if the key is not found, the `int.Max` is returned